### PR TITLE
Airbyte V2 Destination Migration

### PR DESCRIPTION
**Pull request by Janemesi**
- Updates the macro to incorporate the new airbyte changes on how repeat group data is being processed
- Updates the sources yaml to reference the sources destination schema (airbyte_v2)

### CODE REVIEW

File: models/staging/a_sources.yml [hunk 1]
```diff
@@ -29,11 +29,8 @@ sources:
         - name: country_hh_size
         - name: vte_groups_legacy
         - name: land_area_legacy
-        - name: beneficiaries_list
```
Comment (JoeOToole73) [inline models/staging/a_sources.yml:32]: Why have these surveys been removed?

File: models/staging/a_sources.yml
Comment (JoeOToole73) [inline models/staging/a_sources.yml:1]: Can someone confirm whether any surveys which were previously included have been removed during this reworking of the sources list? If so, why is that?

Comment (Janemesi) [reply models/staging/a_sources.yml:1]: @JoeOToole73 Please note that some of the enterprise and land surveys with repeat groups, previously listed under the onadata schema, have been moved to the airbyte_v2 schema. Consequently, they have been removed from the list under the onadata schema in the sources YAML. However, this does not mean they have been omitted from the analysis.
Additionally, the list of sources is exhaustive, but not all sources have been added to the sources YAML. If you come across a survey that appears to be omitted from the analysis, please raise it with us.

**Review by JoeOToole73 — APPROVED** [review]
As I can identify the rough purposes of the changes made, and have had my comment about source maintenance addressed, I am happy to approve and merge these changes. If survey ingestion continues not to function, we will require further assistance to rectify the problem as a priority.

**Pull request merged**